src/app/dashboard-widgets/add-codebase-widget/codebase-item/codebase-item.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="list-group-item list-view-pf-stacked list-view-pf-top-align">
+  <div class="list-view-pf-checkbox f8-card-list-view-pf-checkbox-icon">
+    <span class="avatar fa" [ngClass]="{'fa-github fa-2x': codebase.attributes.type === 'git'}"></span>
+  </div>
+  <div class="list-view-pf-main-info f8-card-list-view-pf-main-info">
+    <div class="list-view-pf-body">
+      <div class="list-view-pf-description">
+        <div class="list-group-item-heading f8-card-list-group-item-heading">
+          <a href="{{codebase.attributes.url}}" target="_blank" class="f8-card-codebase-url truncate">{{codebase.name}}</a>
+        </div>
+        <div class="list-group-item-text f8-card-list-group-item-text">
+          codebase last updated 03/12/2018
+          <!-- replace with real code -->
```

Comment:
Hmm.. do you think this could be replaced now? Otherwise I'd prefer it hidden rather than using a placeholder.

Reply:
Not sure, I intentionally just copied the component verbatim since that part is out of scope of this feature addition. FWIW looking at the `Codebase` type I don't see a property for "last updated". Perhaps it's meant to be the `codebase.gitHubRepo.pushedAt`? I'm not sure what the expectation for the "last updated" item is supposed to be and if a push event is the only thing that would be included.

Reply:
Hmm maybe it's `pushedAt`? No idea; could you comment the whole div?

Reply:
At least for my codebases in my test-space, the `gitHubRepo` property is undefined and so `pushedAt` is not there either. I'll file a separate issue to fix that placeholder and address it separately.

Reply:
https://openshift.io/openshiftio/Openshift_io/plan/detail/542